lnrpc/switchrpc/config_active.go

```diff
@@ -31,4 +32,23 @@ type Config struct {
 	// MacService is the main macaroon service that we'll use to handle
 	// authentication for the Switch rpc server.
 	MacService *macaroons.Service
+
+	// HtlcDispatcher provides the means by which payment attempts can
+	// be dispatched.
+	HtlcDispatcher routing.PaymentAttemptDispatcher
```

Comment:
Then i dont think we need the raw `Switch` above?

Reply:
Indeed the `HtlcDispatcher ` was added to provide limited Switch access. The entire Switch pointer was from modeling the Switch RPC server off the Router RPC server. I'll remove the full Switch access for now. We can add it back in, if it ever becomes necessary.